src/Extensions/Subsites/IndexConfigurationExtension.php

```diff
@@ -15,14 +15,32 @@ class IndexConfigurationExtension extends Extension
     public function updateIndexesForDocument(DocumentInterface $doc, array &$indexes): void
     {
         $docSubsiteId = null;
+        $dataObj = $doc->getDataObject();
```

Comment:
I think moving this outside the `if` has broken the tests, as it's trying to do something odd with the `IndexConfigurationFake` - because now that we are requiring subsites, it always applies this extension, where it wasn't before.

Reply:
Nah, did not work.